# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/midi/tests/browser.ini
# Commit: 225adb498b09
# Full Hash: 225adb498b091a93ddf7081c1f4356f705594a7b
# Author: Gabriele Svelto <gsvelto@mozilla.com>
# Date: 2022-04-12 15:57:11
# Regressor Bug: 1758468
# File Overlap Count: 1
# Description:
#   Bug 1758468 - Use stable, anonymous IDs for Web MIDI ports r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142550
# ==============================================================================

diff -r 3c6e6d0d7b38 -r 225adb498b09 dom/midi/tests/browser.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/midi/tests/browser.ini	Tue Apr 12 12:31:27 2022 +0000
@@ -0,0 +1,12 @@
+[DEFAULT]
+prefs =
+  dom.webmidi.enabled=true
+  midi.testing=true
+  midi.prompt.testing=true
+  media.navigator.permission.disabled=true
+
+[browser_stable_midi_port_ids.js]
+run-if = (os != 'android')
+support-files =
+  port_ids_page_1.html
+  port_ids_page_2.html